train.py

```diff
@@ -129,11 +125,13 @@ def train_gpu(rank,world_size):
 
 
 if __name__ == '__main__':
-    signal.signal(signal.SIGINT, signal_handler) #to really kill the process
```

Comment:
I guess this sneaked in previous PR but is not needed anymore?

Reply:
Actually it's a mistake, we should keep it.